### DISCUSSION TITLE
Save OpenCPN and SignalK settings?

### DISCUSSION THREAD

**Discussion by wsturner88** [bareboat-necessities/lysmarine_gen#236] [Q&A] [answered]
Is there a way to save the configuration of our OpenCPN and signalK so that we can quickly get up and running as we install your latest images?

Thanks!   BTW - Great work, I am starting to really like your package over OpenPlotter..

**Comment by mgrouch** [comment, marked as answer]
Create a gihub repository and store your configuration in it. Write a simple script to restore your configuration from your github repo.
